app/helpers/sync.js

```diff
@@ -32,9 +32,10 @@ export function getTotalActionsTodoCount(state) {
   ];
   const actionsPendingCount = actionsPending.reduce((ac, next) => (next ? ac + 1 : ac), 0);
```

Comment:
We missed adding the state.layers.sync flag so the syncing modal closes before finishing and opens up again.